packages/components/src/Text/Text.mdx

```diff
@@ -69,3 +58,26 @@ only be used right after a page title.
     Drag to rearrange the order that fields show up in Jobber
   </Text>
 </Playground>
+
+## Sizes
+
+## Intro
```

Comment:
Let's bump these size subheaders down to an h3 - I think given the changes we want to go

### Large
(example)
### Base
(example)
### Small
(example)